Backport from plucky to oracular

athos-ribeiro opened the pull request:
Backport from plucky to oracular

PPA: https://launchpad.net/~athos-ribeiro/+archive/ubuntu/ubuntu-container-stack/+packages

DEP8: pending (will run when we get all the containerstack backported in the PPA above).

lvoytek reviewed (approved):
The changes look good to me :)
Everything matches up with plucky other than the continued use of golang 1.22, and changelog reflects that. I noticed that Hlib released a security update last week for all supported releases and plucky though. Should that be included too?

lvoytek commented on debian/changelog:
```diff
@@ -1,8 +1,22 @@
-  * Backport from plucky to oracular.
+  * Backport from plucky to oracular (LP: #2085187)
+  * d/control: build with golang 1.22 for consistency across the supported
+    releaes.
```
```suggestion
    releases.
```

athos-ribeiro commented:
Thanks, Lena! I will re-work all PRs for this package so we stick in v1.7.x for now!

athos-ribeiro commented:
@lvoytek, actually, for oracular only, we should proceed with backporting the plucky version.

The reason is that oracular is already in containerd-app v2. I fixed this PR with addressing your comments (typo + security patch).

I will change all other branches to update to 1.7.x though.
